src/components/Message/MessageSimple/MessageContent.tsx

```diff
@@ -211,16 +226,23 @@ const MessageContentWithContext = <
     reactionPickerVisible,
     reactionsEnabled = true,
     repliesEnabled = true,
-    retrySendMessage,
```

Comment:
do we want to do a similar thing here for what you did for Attachment so we support it as a prop or from context?

Reply:
since this is available from context and isn't something that's customized not sure there's any benefit in passing it as props

Reply:
if it was a prop, clients had the ability to customize it. so some may have use it for something like an encrypted retry send message method which will no longer work without them switching to add this at the context level. I'm fine with that we just are introducing breaking changes of having to use context instead of the gradual change of allowing prop drilling overrides and context and then eventually getting rid of the prop drilling ones

Reply:
same as the other one, it wasn't a true prop because it was only being pulled off props because this was a WithContext component and it's parent was pulling it out of context